sites/public/pages/eligibility/persons.tsx

```diff
@@ -45,12 +45,11 @@ const EligibilityBedrooms = () => {
     eligibilityRequirements.setCompletedSections(CURRENT_PAGE + 1)
   }
 
-  const bedroomsOptions = [
-    { id: "studio", label: t("eligibility.bedrooms.studio") },
-    { id: "oneBdrm", label: "1" },
-    { id: "twoBdrm", label: "2" },
-    { id: "threeBdrm", label: "3" },
-    { id: "fourBdrm", label: "4+" },
+  const personsOptions = [
+    { id: "one", label: "1" },
+    { id: "two", label: "2" },
+    { id: "three", label: "3" },
+    { id: "four", label: "4+" },
   ]
```

Comment:
The mocks have a drop-down. We should also have options up to 8 people, because that's what the AMI calculations are based on.

Reply:
updated to Select/Drop down